src/registries/chart_types.ts

```diff
@@ -184,6 +189,22 @@ chartRegistry.add("scatter", {
   name: _t("Scatter"),
   sequence: 60,
 });
+chartRegistry.add("waterfall", {
+  match: (type) => type === "waterfall",
+  createChart: (definition, sheetId, getters) =>
+    new WaterfallChart(definition as WaterfallChartDefinition, sheetId, getters),
+  getChartRuntime: createWaterfallChartRuntime,
+  validateChartDefinition: (validator, definition: WaterfallChartDefinition) =>
+    WaterfallChart.validateChartDefinition(validator, definition),
+  transformDefinition: (
+    definition: WaterfallChartDefinition,
+    executed: AddColumnsRowsCommand | RemoveColumnsRowsCommand
+  ) => WaterfallChart.transformDefinition(definition, executed),
+  getChartDefinitionFromContextCreation: (context: ChartCreationContext) =>
+    WaterfallChart.getDefinitionFromContextCreation(context),
```

Comment:
More like a question, but can't we write it like
```js
validateChartDefinition: WaterfallChart.validateChartDefinition,
transformDefinition: WaterfallChart.transformDefinition,
getChartDefinitionFromContextCreation: WaterfallChart.getDefinitionFromContextCreation,
```
to avoid using lamda ?

Reply:
Well yes, that's way shorter. I just copy/pasted it from the other chart registry entries. Not sure why we weren't doint it for the others 🤔